website/static/js/institutionProjectSettings.js

```diff
@@ -5,102 +5,174 @@ var Raven = require('raven-js');
+        bootbox.dialog({
+                title: self.pageTitle(),
+                onEscape: function () {
                 },
-                fields: {xhrFields: {withCredentials: true}}
-            }
-        ).done(function (response) {
-            var indexes = self.availableInstitutions().map(function(each){return each.id;});
-            var index = indexes.indexOf(self.selectedInstitution());
-            var added = self.availableInstitutions().splice(index, 1)[0];
-            self.availableInstitutions(self.availableInstitutions());
-            self.affiliatedInstitutions().push(added);
-            self.affiliatedInstitutions(self.affiliatedInstitutions());
-            self.showAdd(false);
-        }).fail(function (xhr, status, error) {
-            $osf.growl('Unable to add institution to this node. Please try again. If the problem persists, email <a href="mailto:support@osf.io.">support@osf.io</a>');
-            Raven.captureMessage('Unable to add institution to this node', {
-                extra: {
-                    url: url,
-                    status: status,
-                    error: error
+                backdrop: true,
+                closeButton: true,
+                message: '<div class="row">  ' +
+                    '<div class="col-md-12"> ' +
+                    '<span>' + message + '</span> ' +
+                    '<div class="radio" > <label for="selectOne"> ' +
+                    '<input type="radio" id="selectOne" type="radio" name="radioBoxGroup"' +
+                    ' value="false" checked="checked"> ' + addToOneMessage + ' </div></label> ' +
+                    '<div class="radio"> <label for="selectAll"> ' +
+                    '<input type="radio" id="selectAll" type="radio" name="radioBoxGroup" value="true"> ' +
+                    addToAllMessage + ' </label> ' + '</div>' + '</div>',
+                buttons: {
+                    cancel: {
+                        label: 'Cancel',
+                        className: 'btn-default',
+                        callback: function () {
+                        }
+                    },
+                    success: {
+                        label: self.buttonInfo().buttonLabel,
+                        className: self.buttonInfo().buttonColor,
+                        callback: function () {
+                            self._modifyInst(item);
+                        }
+                    }
                 }
-            });
+            }
+        ).on('shown.bs.modal', function(e) {
+            if($('input:radio[name=radioBoxGroup]').length) {
+                $('input:radio[name=radioBoxGroup]').click(function() {
+                    self.modifyChildren($(this).val());
+
+                });
+            }
         });
     };
+
+    self.pageTitle = ko.computed(function () {
+        return self.isAddInstitution() ? 'Add institution' : 'Remove institution';
+    });
+
+    self.hasThingsToAdd = ko.computed(function () {
```

Comment:
Minor: This is probably unnecessary. You can just check 

```js
if (self.availableInstitutions.length)
```

where appropriate.